deploy-packit-pr/README.md

```diff
@@ -0,0 +1,120 @@
+# Deploy a testing instance for Packit-as-a-Service
+
+AC:
+
+* figure out how we want to do this epic
+* write down your findings in a document
+* discuss with the team and get a consensus
+
+## How should the deployment be triggered?
+
+At least in the beginning we might not want to deploy every single PR, so we
+might need to have an explicit way for triggering this.
+
+**Minimal solution:** write a script taking some arguments, which each of us
+can run locally. For this each of us would need to have the credentials to the
+environment(s) the script needs to interact with.
+
+**One step further:** have some service/bot, listening to GitHub
+comments/label changes and running the deployment script. We could check with
+softwarefactory folks if we could get a new Zuul pipeline to handle this.
```

Comment:
also github actions could be used

Reply:
but the thing is that we need to share all our secrets to that environment

Reply:
made a note about this.